lib/g_searcher_web/controllers/helpers/search_helper.ex

```diff
@@ -0,0 +1,60 @@
+defmodule GSearcherWeb.Helpers.SearchHelper do
+  @valid_operators ["title", "url", "top_ads", "regular_ads", "status"]
+
+  @spec parse_query(String.t()) :: {:ok, %{String.t() => String.t()}}
+  def parse_query(query) when is_binary(query) do
+    parsed_query =
+      query
+      |> escape_percentage_sign()
+      |> split_string_by_space(ignore_in_double_quotes: true)
```

Comment:
Will we using `ignore_in_double_quotes` option in the future? 🤔

Reply:
I was using it to be a bit more descriptive. I think the module attribute for the regex makes it's descriptive enough. 👍